Added a "plant DNA manipulator" to Botany backroom

coiax opened the pull request:
On bobdobbington's request, this map change is made.

:cl: coiax, bobdobbington, Core0verload
rscadd: An experimental plant DNA manipulator, based on machinery recovered from ancient seed vaults, has been installed in the Botany department.
/:cl:

> 7:22:23 PM bobdobbington  so, not too long ago coreoverload did a huge
> overhaul and refactor of how plants work
> 7:22:48 PM bobdobbington  breaking out each interesting mutant trait
> into its own gene
> 7:23:10 PM bobdobbington  so, like, the slipperiness of bananas, the
> fluacid content of deathnettles, things like that
> 7:23:34 PM bobdobbington  The idea was to be able to crossbreed
> different types of plants
> 7:23:52 PM bobdobbington  So you could have like say grass tiles that
> glowed like glowshrooms
> 7:24:19 PM bobdobbington  Or glowshrooms that could slip
> 7:24:36 PM bobdobbington  This is the machine that enables the
> manipulation of those traits, which already exist in the code

coiax committed:
Added a "plant DNA manipulator" to Botany backroom

On bobdobbington's request, this map change is made.

:cl: coiax, bobdobbington
rscadd: Added plant DNA manipulator to Botany's backroom
/:cl:

7:22:23 PM bobdobbington  so, not too long ago coreoverload did a huge
overhaul and refactor of how plants work
7:22:48 PM bobdobbington  breaking out each interesting mutant trait
into its own gene
7:23:10 PM bobdobbington  so, like, the slipperiness of bananas, the
fluacid content of deathnettles, things like that
7:23:34 PM bobdobbington  The idea was to be able to crossbreed
different types of plants
7:23:52 PM bobdobbington  So you could have like say grass tiles that
glowed like glowshrooms
7:24:19 PM bobdobbington  Or glowshrooms that could slip
7:24:36 PM bobdobbington  This is the machine that enables the
manipulation of those traits, which already exist in the code

Supermichael777 commented:
its work in progresses and horribly op.

LatD commented:
How about adding it to lavaland botany? There people could test it and play with it freely and not affecting the round.

coiax commented:
Literally made this PR because bobdobbington asked me to.

And that's not a bad idea, Lati, although this is a modern machine, making it de-constructable and movable, and vault creatures are found occasionally by miners. Makes the chance of retrieval a lot lower, depends how op it is?

Core0verload commented:
I coded this one, and I support this change. It needs live testing to determine how OP is it, and we can always nerf the fuck out of it if it becomes a problem.

Add a box of plant gene disks to botanists lockers too. The machine is useless without the disks.

coiax commented:
Can you suggest some sane research levels?

Core0verload commented:
Biology 3 and Programming 4.

coiax committed:
Added plant gene disks to secure botany lockers

The Botany plant DNA machine isn't useful without the diskettes. So here
they are.

coiax committed:
Added plant DNA circuit board to imprinter designs

As above.

coiax committed:
Added plant data disks to Protolathe

Print out some botany data disks if they ask nicely.

coiax committed:
Added machinery to lavaland seed vault

Added a condimaster, the new plant DNA manipulator, and a box of plant
DNA diskettes.

Cheridan commented:
>How about adding it to lavaland botany? There people could test it and play with it freely and not affecting the round.

Let's have the actual botanists have something nice for a change instead of giving all their toys to roundstart observers

coiax commented:
@Cheridan too late, already added it the seed vault.

KorPhaeron commented:
@Cheridan 

It's in both

KorPhaeron commented on code/modules/research/designs/machine_designs.dm:
```diff
@@ -471,4 +471,14 @@
+	id = "plantgenes"
+	req_tech = list("programming" = 4, "biotech" = 3)
+	build_type = IMPRINTER
+	materials = list(MAT_GLASS = 1000, "sacid" = 20)
```
This is basically free, but I guess these are placeholder/test values anyway

coiax replied:
It's a circuit board, the cost is standardised.

KorPhaeron commented on code/modules/research/designs/machine_designs.dm:
```diff
@@ -471,4 +471,14 @@
+
+/datum/design/plantgenes
+	name = "Machine Design (Plant DNA Manipulator Board)"
+	desc = "The circuit board for a grounding rod."
```
Wrong desc

coiax committed:
Fixed inaccurate description of design item
